stories/studies/StudySet.svelte

```diff
@@ -0,0 +1,76 @@
+<script>
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+  import StudyCard from "../../src/components/study-card/StudyCard.svelte";
+</script>
+
+{#each [false, true] as joined}
+  <StudyCard
+    {joined}
+    endDate={new Date(2021, 5, 5)}
+    joinedDate={new Date(2020, 12, 5)}
+    imageSrc="https://addons.cdn.mozilla.net/static/img/addon-icons/default-64.png"
+    dataCollectionDetails={['the date and time', 'number of browser tabs open']}
+    tags={['artificial intelligence', 'misinformation']}
+    privacyPolicyLink={'/'}>
+    <span slot="name">Smoke Test Example Study 1</span>
+    <span slot="author">The Mozilla Foundation</span>
+    <p slot="description">
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+      tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+      veniam, ut enim ad minim veniam, ut enim ad minim veniam.
```

Comment:
Do we have an issue file for changing this?

Reply:
Yes, I believe we do.